modules/test/playwright/fixtures/login.fixture.ts

```diff
@@ -0,0 +1,107 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import {Cookie, Page, expect, test} from '@playwright/test';
+
+import {liferayConfig} from '../liferay.config';
+import {createTempFile, readTempFile} from '../utils/util';
+
+export interface Login {
+	password: string;
+	sessionId: string;
+	user: string;
+}
+
+const TIMEOUT_OPTION = {
+	timeout: 5 * 1000,
```

Comment:
nit: sometimes it's helpful to put this as a named constant `const FIVE_SECONDS = 5 * 1000`